x-pack/plugins/uptime/public/components/overview/alerts/monitor_expressions/filters_expression_select.tsx

```diff
@@ -123,43 +122,60 @@ export const FiltersExpressionsSelect: React.FC<Props> = ({
     (curr) => curr.selectedItems.length > 0 || newFilters?.includes(curr.fieldName)
   );
 
+  const indexPattern = useIndexPattern();
+
   return (
     <>
-      {filtersToDisplay.map(({ description, value, ...item }) => (
-        <EuiFlexGroup key={item.id}>
-          <EuiFlexItem>
-            <FilterPopover
-              {...item}
-              btnContent={
-                <EuiExpression
-                  aria-label={item['aria-label']}
-                  color={'secondary'}
-                  data-test-subj={'uptimeCreateStatusAlert.' + item.id}
-                  description={description}
-                  value={value}
-                  onClick={() => setIsOpen({ ...isOpen, [item.id]: !isOpen[item.id] })}
+      {filtersToDisplay.map(
+        ({ description, id, title, value, fieldName, ariaLabel, selectedItems }) => (
+          <EuiFlexGroup key={id}>
+            <EuiFlexItem>
+              {indexPattern && (
+                <FieldValueSuggestions
+                  filters={[]}
+                  key={fieldName}
+                  indexPatternTitle={indexPattern.title}
+                  sourceField={fieldName}
+                  label={title}
+                  onChange={(vals) => {
```

Comment:
FieldValueSuggestions is a pretty heavy weight component. We may benefit from memoization here.